config/config_params_test.go

```diff
@@ -427,6 +435,90 @@ var _ = Describe("DatastoreConfig tests", func() {
 			Expect(c.DatastoreConfig().Spec.K8sDisableNodePoll).To(BeTrue())
 		})
 	})
+	// Note that these tests set common calico env variables, as there's no other way to test that the datastore config loads
+	// the configuration from the apiconfig and then overwrites them with the felix config. They shouldn't persist between
+	// tests, but this is worth noting that this is happening and there isn't another way to properly test this functionality
+	// without using the environment variables.
```

Comment:
This comment isn't correct now.

Reply:
No it isn't correct, thanks for pointing that out.